### ISSUE TITLE
4953 WHQL drivers removed Fan controls ...

### ISSUE THREAD

**Issue by freak2fast4u** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#603]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Intel Arc Control

### Processor / Processor Number [Required]

N/A

### Graphic Card [Required]

Intel Arc A770 16GB LE

### GPU Driver Version [Required]

31.0.101.4953

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build Number [Required]

- [ ] Windows 11 23H2
- [ ] Windows 11 22H2
- [ ] Windows 11 21H2
- [X] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

N/A

### Description and steps to reproduce [Required]

![image](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/3486369/bc578fb9-64fb-44f4-a7b2-665b28dabcab)

**Comment by IGCIT** [comment]
hi,

could be the same issue as #413 

tl;dr
DDU install and reboot your system

**Comment by freak2fast4u** [comment]
So ... installing 4953 (with the "Clean install" box checked) over 4952 looses fan controls, but DDU'ing the drivers and reinstalling 4953 restores the fan controls (confirmed just now). 

I shouldn't have to mess with DDU at every single driver update !

I wonder how it went for those using Intel DSA, but my guess is their experience was the same, and it leaves a sour taste every single time.

**Comment by Nidzhun** [comment]
I removed old driver with ddu and after installing 4953 i could not find fan control in arc software and could not change performance settings with some warning in windows notifications. After 3 or 4 reboots performance settings with fan control started working; seems that it is software problem with driver initialising or with arc software itself.

**Comment by Karen-Intel** [comment]
Hi all, I have just updated my machine to v.4953 and no issues with fan controls.
About DDU, it is strongly recommended to use it as there can be big differences between WHQL and non-WHQL versions

![image](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/91162218/3ba48e40-e808-4305-b444-b878fa8e6805)

Karen

**Comment by Arturo-Intel** [comment]
@freak2fast4u It is really strange behavior indeed!
Based on the comments from @Nidzhun is not related to DDU prior install the latest driver.
We will need to dig on this a little bit more in our labs.

We will get back to you guys if we need more info to replicate this behavior.

--r2

**Comment by Squall-Leonhart** [comment]
> it is strongly recommended to use it as there can be big differences between WHQL and non-WHQL versions

It is strongly recommended you get your upgrade process down,  DDU is a tool that can damage a system when over used and carries the same warnings as the Driver Cleaner of the 2000's.

**Comment by freak2fast4u** [comment]
Well, something is definitely wrong with the driver installer IMHO. I've been running 4972 beta for a few weeks after a clean DDU reinstall, so far so good. Yesterday I installed Assasin's Creed Syndicate and tried to run it => it crashed during the intro video, every single time. So I'm here thinking maybe the beta drivers broke something, so I DDU'd back to 4953 WHQL ... and guess what ? It stopped crashing. But I had a hunch, so I DDU'd the drivers back to 4972 beta again, and the crash didn't reoccur. What the actual F* ? My driver install was clean to begin with and wasn't tampered with for the past several weeks, but somehow reinstalling them resolved a crash ? 

The one single thing I did differently this time, was to uncheck the "Launch Arc Control" option before clicking on "Restart required" at the end of the driver install procedure. After all, launching Arc Control just before rebooting makes no sense to begin with, but since it was checked by default I took the habit of leaving it ... until now. Maybe the reboot process aborts the first Intel Arc Control startup (post-install) and leaves it in a random unstable state ?

**Comment by Gabriela-Intel** [comment]
Let's close out this issue since its originally reporting that fan controls are missing and it's been confirmed they are indeed there using the latest drivers. Please open a new issue if there's anything else requiring investigation. Thanks all! 

@IGCIT
